tests/integration/services/user.test.ts

```diff
@@ -431,6 +442,109 @@ describe('user service', () => {
             const userUpdated = await UserService.edit(address, data);
 
             expect(userUpdated).to.include(data);
-        })
-    })
+        });
+    });
+
+    describe('delete', () => {
+        let users: User[];
+        let managers: ManagerAttributes[];
+        let communities: CommunityAttributes[];
+
+        before(async () => {
+            users = await UserFactory({ n: 4 });
+            communities = await CommunityFactory([
+                {
+                    requestByAddress: users[0].address,
+                    started: new Date(),
+                    status: 'valid',
+                    visibility: 'public',
+                    contract: {
+                        baseInterval: 60 * 60 * 24,
+                        claimAmount: '1000000000000000000',
+                        communityId: 0,
+                        incrementInterval: 5 * 60,
+                        maxClaim: '450000000000000000000',
+                    },
+                    hasAddress: true,
+                },
+            ]);
+            managers = await ManagerFactory(
+                [users[0]],
+                communities[0].publicId
+            );
+        });
+
+        it('should not delete a user when he is one of the only two managers in the community', async () => {
+            UserService.delete(users[0].address)
+                .catch((err) => expect(err).to.be.equal('Not enough managers'))
+                .then(() => {
+                    throw new Error('expected to fail');
+                });
+        });
+
+        it('manager should be target to be delete', async () => {
+            await ManagerFactory([users[2], users[3]], communities[0].publicId);
+
+            await UserService.delete(users[0].address);
+
+            const user = await models.user.findOne({
+                where: {
+                    address: users[0].address,
+                },
+            });
+
+            expect(user?.deletedAt).to.be.not.null;
+        });
+
+        it('should not delete a user when he is one of the only two managers that are not in a deletion process', async () => {
```

Comment:
for simplicity/readability
```suggestion
        it('should not delete with only two managers (not in deletion process)', async () => {
```